### PR TITLE
Nanotrasen Prometheus-Class Plasma Tanker

### PR DESCRIPTION
**Pull request by BBBBAface**
## About The Pull Request

Shiptest didn't want it so here is a ship I spent about 20 hours working on. There weren't any ships like it on Shiptest; I'm not sure if there are any plasma tankers on this codebase or not, but here's the fully functioning ship.

This PR adds a new Nanotrasen ship to the game, A plasma transport vessel with limited facilities but enough plasma to last a large settlement indefinitely.

Specs:
Limited to 2 per round
440Ton weight with 6 ion engines
Supports a crew of up to 7, but works best with 3 - 5 Power-management system 
Around 2,500,000 MOL of plasma gas
5,000cr starting cash
Stocked medbay
Extremely limited weaponry
Basic mining gear
Working Exterior / Engine / Bay shutters and holofans
"Robust" atmos setup
Outpost comms console

Pros: 
• You will never run out of plasma gas
• Defribulator and surgical tools
• A cell for troublesome patients
• Engineering protolathe
• Large bay doors and engineering area
• Easily customizable
• Only ship of its kind
• Slot machine!
• Shoilet!

Cons:
• NO R&D capabilities
• Limited security gear
• NO way of actually utilizing the plasma it has stored
• Only 3 hardsuits
• No launchpad or outpost trading console
• Slightly cramped design
• No insulated gloves; you're going to have to drink Rubbernecks

- [X] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game

Currently, no ships are designed to trade gas, leaving the niche unfilled. This is the solution.

This ship would be a decent trading vessel, and because of the defibrillator, it could also make a good emergency response vessel, able to refuel ships and revive the crew. Lack of plasma engines or TEG means the plasma gas is useless on round start. Trading with other players is required due to the lack of research capabilities.

## Changelog

:cl:
Add: Prometheus-class tanker
/:cl:

### CODE REVIEW

**Comment by MoonFalcon** [comment]
Looks like it's failing linting. Really like the ship by the way

**Comment by HourlyLimit** [comment]
Nice ship i hope to see it ingame no idea why shiptest would want to give this up.

**Comment by HourlyLimit** [comment]
heres the error something about an id 
image

**Comment by Jackriip** [comment]
Looks great! I'd be happy to test merge it if you get the linting fixed

**Comment by BBBBAface** [comment]
> Nice ship i hope to see it ingame no idea why shiptest would want to give this up.

they said to wait for a rework to "the gas economy" and redo the pr when that's done, so basically just wait indefinitely.

File: _maps/Configs/nanotrasen_tanker.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"prefix": "NTSV",
```
Comment (GuiltyNeko) [inline _maps/Configs/nanotrasen_tanker.json:3]: This will need to be "NEU", as, tragically, we only have three accepted prefixes here.

**Review by GuiltyNeko — CHANGES_REQUESTED** [review]
Window spawners spawn grilles under them, you should remove the grilles on the same tiles as your window spawners. In addition, you should put the ID back on the door, the linters just didn't like the ID var on the docking port. I'd also like you to replace the bridge holopad with a holopad/emergency/command, as most/all of our ships have one and that one in particular can help with access issues faced by a crew. And if I remember correctly, cyclelink helpers only work on doors in cardinal directions from each other. If you want the airlock to cycle in that way you'll need cyclelink_helper_target.

**Comment by BBBBAface** [comment]
> Window spawners spawn grilles under them, you should remove the grilles on the same tiles as your window spawners. In addition, you should put the ID back on the door, the linters just didn't like the ID var on the docking port. I'd also like you to replace the bridge holopad with a holopad/emergency/command, as most/all of our ships have one and that one in particular can help with access issues faced by a crew. And if I remember correctly, cyclelink helpers only work on doors in cardinal directions from each other. If you want the airlock to cycle in that way you'll need cyclelink_helper_target.

The airlock is fully functional as-is, this ship has been playtested quite a bit. I'm fixing the issues and updating the branch.

**Comment by BBBBAface** [comment]
I put the id back on the door and it returns an error again, should I just delete it?

**Comment by GuiltyNeko** [comment]
Ah, on doors, the proper var name is id_tag. That should do it.

**Review by GuiltyNeko — APPROVED** [review]
I'll TM this on the next round of testmerges

**Comment by BBBBAface** [comment]
fixing my GitHub stuffi messed up awhile back, im just going to redo this pr

**Pull request closed without merging**